src/main/java/com/puppycrawl/tools/checkstyle/DefaultLogger.java

```diff
@@ -159,30 +166,26 @@ public void addError(AuditEvent event) {
     @Override
     public void addException(AuditEvent event, Throwable throwable) {
         synchronized (errorWriter) {
-            final Violation addExceptionMessage = new Violation(1,
-                Definitions.CHECKSTYLE_BUNDLE, ADD_EXCEPTION_MESSAGE,
-                new String[] {event.getFileName()}, null,
-                Violation.class, null);
-            errorWriter.println(addExceptionMessage.getViolation());
+            final LocalizedMessage addExceptionMessage = new LocalizedMessage(
+                    new String[] {event.getFileName()}, ADD_EXCEPTION_MESSAGE);
+            errorWriter.println(addExceptionMessage.getMessage());
             throwable.printStackTrace(errorWriter);
         }
     }
 
     @Override
     public void auditStarted(AuditEvent event) {
-        final Violation auditStartMessage = new Violation(1,
-            Definitions.CHECKSTYLE_BUNDLE, AUDIT_STARTED_MESSAGE, null, null,
-            Violation.class, null);
-        infoWriter.println(auditStartMessage.getViolation());
+        final LocalizedMessage auditStartMessage = new LocalizedMessage(
+                AUDIT_STARTED_MESSAGE);
```

Comment:
make it single line.

apply this to other cases like this.

Reply:
Done